addons/mail/models/mail_thread.py

```diff
@@ -1817,9 +1817,13 @@ def message_post(self, *,
             if subtype_xmlid in ('mail.mt_note', 'mail.mt_comment'):
                 subtype_id, internal = self.env['ir.model.data'].xmlid_to_res_id(subtype_xmlid), subtype_xmlid == 'mail.mt_note'
```

Comment:
Seems like a dangerous optimization, if someone changed the internal of its default subtypes for some reason...

Reply:
Then everything is fucked up. Comment and note are hardcoded in marble !